lecture_notes/09 functions/2-scope.py

```diff
@@ -0,0 +1,25 @@
+def toLowerCase(cities):
+    answearSheat = []
+    for item in cities:
+        answearSheat.append(item.lower())
+    print(answearSheat)
+def toUpperCase(cities):
+    answearSheat = []
+    for item in cities:
+        answearSheat.append(item.upper())
+    print(answearSheat)
+def findLengthOfString(cities):
+    answearSheat = []
+    for item in cities:
+        answearSheat.append(len(item))
+    print(answearSheat)
+def f1():
+    citiesFirst = ['Athens', 'Roma']
+    toUpperCase(citiesFirst)
+    toLowerCase(citiesFirst)
+    citiesSecond = ['London', 'Beijing', 'Kiev']
+    toUpperCase(citiesSecond)
```

Comment:
- Add single empty line to separate fiunctions
- Use def to create nested contexts